docs/adding_packages/sources_and_patches.md

```diff
@@ -0,0 +1,33 @@
+# Sources and Patches
+
+This documents contains everything related to the `source()`. This includes picking sources, where they should come from and goes into when and how to modify sources.
+This is a very important aspects and it helps us to establish the quality of the packages offered by ConanCenter.
+
+<!-- toc -->
+## Contents
+
+  * [Sources](#sources)<!-- endToc -->
+
+## Sources
+
+**Origin of sources:**
+
+* Library sources should come from an official origin like the library source code repository or the official
+release/download webpage.
+
+* If an official source archive is available, it should be preferred over an auto-generated archive.
```

Comment:
maybe we should explain it a little bit more, on how to distinguish these two.
e.g., in GitHub projects, prefer archives from the `releases` tab, rather `code` - `download zip`